docs/src/main/tut/annotations.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: docs
 title: Annotations
-permalink: /docs/rpc/annotations
+permalink: /annotations
```

Comment:
On root? should it be `/mu/scala/annotations`?

Reply:
That is set through the `urlBase` parameter

Reply:
It's ok, the baseUrl is doing the rest ;)